src/main/java/io/ncbpfluffybear/supserv/items/FarmersHoe.java

```diff
@@ -0,0 +1,61 @@
+package io.ncbpfluffybear.supserv.items;
+
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.protection.ProtectableAction;
+import org.bukkit.CropState;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.Ageable;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+
+public class FarmersHoe  extends SimpleSlimefunItem<ItemUseHandler> {
+
+    public FarmersHoe(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
+        super(category, item, recipeType, recipe, recipeOutput);
+
+    }
+
+    List<Material> farmables = Arrays.asList(Material.DIRT, Material.GRASS_BLOCK, Material.COARSE_DIRT);
+    @Nonnull
+    @Override
+    public ItemUseHandler getItemHandler() {
+        return e -> {
+
+            Player p = e.getPlayer();
+            ItemStack item = e.getItem();
+
+            if (!isItem(item))
```

Comment:
Unnecessary check, since this inside an ItemHandler for the tool it is a given that this item matches. These checks are only necessary in Listener based handlers.